core/src/main/java/io/kestra/core/storages/Storage.java

```diff
@@ -15,6 +16,20 @@
  */
 public interface Storage {
 
+    /**
+     * Gets access to the Key-Value store for the contextual namespace.
+     *
+     * @return The {@link KVStore}.
+     */
+    KVStore namespaceKv();
+
+    /**
+     * Gets access to the Key-Value store for the given namespace.
+     *
+     * @return The {@link KVStore}.
+     */
+    KVStore namespaceKv(String namespace);
+
```

Comment:
I would have put these ones in the RunCOntext interface not the Storage interface because Kv using the internal storage is an implementation detail.

Reply:
I'm not sure we want to expose this in the run context directly as we have tasks for it. Moreover I put this here because it's a Store so Storage felt like the right place (not necessarily thinking about the internal storage, but more the notion of Storage itself)